### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by fboucquez**
Upgraded open api to 0.8.5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
-# nem2-sdk for Java/Kotlin/Scala
+# symbol-sdk for Java/Kotlin/Scala
 #
```
Comment (dgarcia360) [inline README.md]: Remove extra "#" tag

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
-# nem2-sdk for Java/Kotlin/Scala
+# symbol-sdk for Java/Kotlin/Scala
```
Comment (dgarcia360) [inline README.md]: Symbol SDK for Java/Kotlin/Scala

File: README.md [hunk 3]
```diff
@@ -33,76 +34,76 @@ OR
...
-- **sdk-vertx-client:** The nem2-sdk-java Implementation that uses Vertx and generated `api-vertx-client` lib and dtos. A client may depend on this SDK implementation if Vertx is the selected implementation (e.g. server users).
-- **sdk-okhttp-client:** The nem2-sdk-java Implementation that uses OkHttp and the generated `api-okhttp-gson-client`. A client may depend on this SDK implementation if OkHttp is the selected implementation (e.g. android users).
+- **sdk-vertx-client:** The symbol-sdk-java Implementation that uses Vertx and generated `symbol-openapi-vertx-client` lib and dtos. A client may depend on this SDK implementation if Vertx is the selected implementation (e.g. server users).
+- **sdk-okhttp-client:** The symbol-sdk-java Implementation that uses OkHttp and the generated `symbol-openapi-okhttp-gson-client`. A client may depend on this SDK implementation if OkHttp is the selected implementation (e.g. android users).
 - **integration-tests:** This module is in charge of running integration tests against all implementations. The integration tests exercise how the implementation work against a given catapult server.
 
 
-## nem2-sdk Releases
+## symbol-sdk Releases
```
Comment (dgarcia360) [inline README.md]: `## Releases`

File: README.md [hunk 4]
```diff
@@ -33,76 +34,76 @@ OR
...
+This project is developed and maintained by NEM Foundation. Contributions are welcome and appreciated. You can find [symbol-sdk on GitHub][self];
 Feel free to start an issue or create a pull request. Check [CONTRIBUTING](CONTRIBUTING.md) before start.
 
 ## Getting help
 
 We use GitHub issues for tracking bugs and have limited bandwidth to address them.
 Please, use the following available resources to get help:
 
-- [nem2-cli documentation][docs]
+- [symbol-cli documentation][docs]
```
Comment (dgarcia360) [inline README.md]: Change for `Symbol SDK Java Reference` and update link.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,14 @@
...
-[![Coverage Status](https://coveralls.io/repos/github/nemtech/nem2-sdk-java/badge.svg?branch=master)](https://coveralls.io/github/nemtech/nem2-sdk-java?branch=master)
+[![symbol-sdk-vertx-client](https://maven-badges.herokuapp.com/maven-central/io.nem/symbol-sdk-vertx-client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.nem/symbol-sdk-vertx-client)
+[![symbol-sdk-okhttp-client](https://maven-badges.herokuapp.com/maven-central/io.nem/symbol-sdk-okhttp-client/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.nem/symbol-sdk-okhttp-client)
+[![Build Status](https://api.travis-ci.org/nemtech/symbol-sdk-java.svg?branch=master)](https://travis-ci.org/nemtech/symbol-sdk-java)
+[![Coverage Status](https://coveralls.io/repos/github/nemtech/symbol-sdk-java/badge.svg?branch=master)](https://coveralls.io/github/nemtech/symbol-sdk-java?branch=master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 
-The official nem2-sdk for Java, Kotlin and Scala to work with the NEM2 (a.k.a Catapult).
+The official symbol-sdk for Java, Kotlin and Scala to work with the SYMBOL (a.k.a Catapult).
```
Comment (dgarcia360) [inline README.md]: Replace with ``The Symbol SDK for Java, Kotlin and Scala.``

File: README.md (hunk 1 above)
Comment (fboucquez) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (fboucquez) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (fboucquez) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (fboucquez) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (fboucquez) [reply README.md]: which link?

Comment (fboucquez) [reply README.md]: [sdk-ref]?
javadocs?

Comment (dgarcia360) [reply README.md]: Yes, javadocs

**Pull request merged**